src/sentry/replays/consumers/recording_buffered.py

```diff
@@ -195,12 +191,19 @@ def new(self) -> RecordingBuffer:
 
 def process_message(buffer: RecordingBuffer, message: bytes) -> None:
     with sentry_sdk.start_span(op="replays.consumer.recording.decode_kafka_message"):
-        decoded_message: ReplayRecording = RECORDINGS_CODEC.decode(message)
+        try:
+            decoded_message: ReplayRecording = RECORDINGS_CODEC.decode(message)
+        except ValidationError:
+            # TODO: DLQ
+            return None
 
     try:
         headers, recording_data = process_headers(decoded_message["payload"])
-    except MissingRecordingSegmentHeaders:
-        logger.warning("missing header on %s", decoded_message["replay_id"])
+    except Exception:
```

Comment:
is Exception too general of a catch here?

Reply:
Typically I'd say yes.  But if its going to crashloop the consumer and break ingestion then I suppose no?  I think emitting the handled exception to sentry would allow us to monitor it at least so I'll make that change.